index.html

```diff
@@ -74,12 +66,35 @@ <h1>I’m Stanley.<br>Glad to see you!</h1>
             </ul>
             </div>
         </section>
+    </header>
 
+    <menu class="menu menu-visibility">
+        <div class="cancel-icon" id="cancel-icon">
+            <img src="./assets/IconCancel.svg" alt="">
+        </div>
+        <ul class = "menuList">
+            <li class="menuLi"><a href="#" data-target="cards">Portfolio</a></li>
+            <li class="menuLi"><a href="#" data-target="about">About</a></li>
+            <li class="menuLi"><a href="#" data-target="contact">Contact</a></li>
+        </ul>
+    </menu>
+
+  
+    <!-- For mobile -->
+    <section class="popup-hero" id="cont"></section>
+    <!-- For desktop -->
+     <section class="D-popup-hero" id="D-cont">
+    </section>
```

Comment:
- [ ] Good that you have created your popup windows dynamically using javascript :+1: But please make sure that you remove the already created HTML code for your projects section and create them dynamically using javaScript with the information from the projects array you have created. :pray:

Reply:
This tag was utilized in my code to dynamically generate strings in the JavaScript code using the DOM's built-in innerHTML function. Only the string would the browser use to display the popup

Reply:
Do the same to display the project cards dynamically. Please that is what I mean